### DISCUSSION TITLE
Lambda UknownError

### DISCUSSION THREAD

**Discussion by Ramdan241004** [awsugid/forum#34] [General]
![IMG_20240718_073651](https://github.com/user-attachments/assets/d772cc83-72d5-4d7c-afd4-111a5eabfaa8)
Izin bertanya kak ada yang tau kah ini kenapa ya? Padahal saya menggunakan akun AWS Root bukan IAM User

**Comment by saiqulhaq** [comment]
coba buka pake laptop, mungkin ada script JS yg gak bisa jalan di chrome mobile
